vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AutoFixShouldNotBeCheckedIn.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ReactNative.Managed.CodeGen.UnitTests
+{
+  [TestClass]
+  public class AutoFixShouldNotBeCheckedIn : CodeGenTestBase
+  {
+    [TestMethod]
+    public void EnsureAutoFixIsFalse()
+    {
+      Assert.IsFalse(AutoFixLkgs, "Do Not check AutoFixLkgs in as true, since it will not catch any failures otherwise.");
```

Comment:
not sure what this file is testing?

Reply:
The base test class has a boolean that you can set to regenerate the lkg's (in case there are a lot of changes in the codegen that touch a lot of tests). This test fails if that flag is turned on. This makes sure that nobody accidentally creates a check-in that sets the flag to true and have all the tests pass form here on out because the LKG matches :)

Reply:
replace "catch any failures otherwise" with that description? :)